workflows/scripts/aggregate_cells.py

```diff
@@ -50,13 +51,13 @@ def aggregate(
         datefmt="%Y-%m-%d %H:%M:%S",
     )
 
-    logging.info(f"Aggregating Single Cells in: {sql_file}")
+    logging.info(f"Aggregating Single Cells in: {str(sqlite_file_path)}")
 
     # loading parameters
     logging.info(f"Loading single cell aggregation configurations from: {config}")
-    aggregate_path_obj = Path(config)
+    aggregate_path_obj = pathlib.Path(config)
     aggregate_config_path = aggregate_path_obj.absolute()
-    config_check = Path(aggregate_config_path).is_file()
+    config_check = pathlib.Path(aggregate_config_path).is_file()
```

Comment:
Consider using `is_file()` directly in the if statement below instead of creating a new variable for one-time use.